diff --git a/dist/utils.js b/dist/utils.js
index 217aef141ae9c0cde9aa05b71d7ba11c3ebe13e9..870c5e142da2a865e9cdcf10e9492efc4768b324 100644
--- a/dist/utils.js
+++ b/dist/utils.js
@@ -9,21 +9,21 @@ exports.normalizeThemeValue = normalizeThemeValue;
 exports.round = round;
 exports.lineHeightProperties = lineHeightProperties;
 function isRelativeValue(value) {
-    let isPercentValue = value.endsWith('%');
+    let isPercentValue = value?.endsWith?.('%');
     let isUnitlessValue = /\d$/.test(value);
     return isPercentValue || isUnitlessValue;
 }
 function getRelativeValue(value) {
-    let isPercentValue = value.endsWith('%');
+    let isPercentValue = value?.endsWith?.('%');
     return isPercentValue
         ? Number.parseInt(value.replace('%', ''), 10) / 100
         : Number.parseFloat(value);
 }
 function normalizeValue(value, root, fs) {
     value = Array.isArray(value) ? value[0] : value;
-    if (value.endsWith('px'))
+    if (value?.endsWith?.('px'))
         return Number.parseFloat(value.replace('px', ''));
-    if (value.endsWith('rem'))
+    if (value?.endsWith?.('rem'))
         return root * Number.parseFloat(value.replace('rem', ''));
     if (isRelativeValue(value) && fs !== undefined) {
         return fs * getRelativeValue(value);
